sympy/core/expr.py

```diff
@@ -3641,17 +3641,8 @@ def round(self, n=None):
         Notes
         =====
 
-        The Python builtin function, round, always returns a
-        float in Python 2 while the SymPy round method (and
-        round with a Number argument in Python 3) returns a
-        Number.
-
-        >>> from sympy.core.compatibility import PY3
-        >>> isinstance(round(S(123), -2), Number if PY3 else float)
-        True
-
-        For a consistent behavior, and Python 3 rounding
-        rules, import `round` from sympy.core.compatibility.
+        When applied to a Number, `round` will return a Number, not a Python
```

Comment:
I think this whole note should be removed. I would automatically expect every SymPy function to return a SymPy type, which is all that this says now.

Reply:
`round` is a Python builtin -- that's why this note is added. It let's the user know that SymPy has defined a method call be `round` so a SymPy Number (not Python number) is returned:
```python
>>> type(round)
<type 'builtin_function_or_method'>
```